docs/command-line-interface.md

```diff
@@ -0,0 +1,266 @@
+# OCI Runtime Command Line Interface
+
+This file defines the OCI Runtime Command Line Interface version 1.0.0.
```

Comment:
Could you maybe add a sentence stating that this interface is specified as a test harness for compliance testing and runtimes wishing to submit for OCI runtime compliance must implement this interface for testing to be possible